demo/main.js

```diff
@@ -0,0 +1,140 @@
+document.body.scrollTop += 1;
+var doc = document.body.scrollTop ? document.body : document.documentElement;
```

Comment:
Tests are failing in your branch because this code doesn't follow [Standard](https://github.com/standard/standard) style. `standard --fix` would help here